hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/ScheduleServiceImpl.java

```diff
@@ -32,4 +43,64 @@ public void registerSchemas(@NonNull final SchemaRegistry registry) {
         registry.register(new V0490ScheduleSchema());
         registry.register(new V0570ScheduleSchema());
     }
+
+    @Override
+    public Iterator<ExecutableTxn> iterTxnsForInterval(
+            final Instant start, final Instant end, final Supplier<StoreFactory> cleanupStoreFactory) {
+        final var store = cleanupStoreFactory.get().writableStore(WritableScheduleStoreImpl.class);
+        // Filter transactions that are not executed/deleted and have all the required keys
```

Comment:
Comment is out of date; this isn't actually filtering by required keys.

Reply:
Fixed.